src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,15 @@
+
+server.port=8443
+
+
+# postgres db connection
+# todo: use environment system variables
+spring.datasource.url=jdbc:postgresql://localhost:5432/arena-db
+spring.datasource.password=@Arena@123%ndg.#
+spring.datasource.username=arena
+spring.datasource.driver-class-name=org.postgresql.Driver
+
```

Comment:
Add this to gitignore, preferably only leave the config keys and omit the values. Not too critical but best practice